### PR TITLE
Add Node.JS Example

### PR DESCRIPTION
**Pull request by mskian**

### CODE REVIEW

File: docs/more-pushmsg.md [hunk 1]
```diff
@@ -83,3 +83,36 @@ switch ($code) {
...
+  priority: 5
+};
+axios({
+  method: "post",
+  headers: {
+    "Content-Type": "application/json"
+  },
+  url: url,
+  data: bodyFormData
+})
```
Comment (eternal-flame-AD) [inline docs/more-pushmsg.md]: Please either use semicolons throughout the file, or don't use them at all. I personally prefer omitting them.

File: docs/more-pushmsg.md [hunk 2]
```diff
@@ -83,3 +83,36 @@ switch ($code) {
...
+// Send Push Message to Gotify Server //
+////////////////////////////////////////
+
+var url = "http://localhost:8008/message?token=<apptoken>";
+var bodyFormData = {
+  title: "Hello from Javascript",
+  message: "Test Push Service from Node.js",
+  priority: 5
+};
+axios({
```
Comment (eternal-flame-AD) [inline docs/more-pushmsg.md]: Should you `require()` axios first?

File: docs/more-pushmsg.md [hunk 3]
```diff
@@ -83,3 +83,36 @@ switch ($code) {
...
+  },
+  url: url,
+  data: bodyFormData
+})
+  .then(function(response) {
+    console.log(response.data);
+  })
+  .catch(function(error) {
+    if (!error.response) {
+      console.log("Gotify API URL is Missing");
```
Comment (jmattheis) [inline docs/more-pushmsg.md]: Is this really true? I think the response is `undefined` if the url that was defined isn't reachable or does not exist.

File: docs/more-pushmsg.md [hunk 4]
```diff
@@ -83,3 +83,36 @@ switch ($code) {
         echo "<strong>Hmm Something Went Wrong or HTTP Status Code is Missing</strong>";
 }
 ```
+
+### Node.js - Using Axios
```
Comment (jmattheis) [inline docs/more-pushmsg.md]: I'd prefer to name this just JavaScript, there is nothing which prevents this from running in a non nodejs environment.
```suggestion
### JavaScript
```

File: docs/more-pushmsg.md (hunk 3 above)
Comment (mskian) [reply docs/more-pushmsg.md]: if URL was not in Correct format 
if not reachable the URL

File: docs/more-pushmsg.md [hunk 5]
```diff
@@ -83,3 +83,36 @@ switch ($code) {
...
+
+```javascript
+////////////////////////////////////////
+// Send Push Message to Gotify Server //
+////////////////////////////////////////
```
[the remark this answers is not in the document]
Comment (mskian) [reply docs/more-pushmsg.md]: sorry  I forget to remove it

File: docs/more-pushmsg.md (hunk 1 above)
Comment (mskian) [reply docs/more-pushmsg.md]: actually it calls from the above values

```js
var url = "http://localhost:8008/message?token=<apptoken>";
var bodyFormData = {
  title: "Hello from Javascript",
  message: "Test Push Service from Node.js",
  priority: 5
};
```

File: docs/more-pushmsg.md (hunk 2 above)
Comment (mskian) [reply docs/more-pushmsg.md]: Nope, actually I use Gotify Notification system on My Node script - https://github.com/mskian/ssl-expiry-reminder/blob/90ec22a15f8fb56c8d18832c7ab8177261f18f61/check.js#L43

File: website/package.json [hunk 6]
```diff
@@ -11,7 +11,7 @@
     "rename-version": "docusaurus-rename-version"
   },
   "devDependencies": {
-    "docusaurus": "^1.8.0",
+    "docusaurus": "^1.13.0",
```
[the remark this answers is not in the document]
Comment (mskian) [reply website/package.json]: actually I automatically replace the dep while `npm audit fix`

File: docs/more-pushmsg.md (hunk 3 above)
Comment (jmattheis) [reply docs/more-pushmsg.md]: Could you `console.log(error)` instead? This should result in a sufficient error message.

File: docs/more-pushmsg.md (hunk 2 above)
Comment (jmattheis) [reply docs/more-pushmsg.md]: In your script you do:
```
const axios = require('axios');
```

File: docs/more-pushmsg.md (hunk 3 above)
Comment (mskian) [reply docs/more-pushmsg.md]: sure let me change

File: docs/more-pushmsg.md (hunk 2 above)
Comment (mskian) [reply docs/more-pushmsg.md]: yes, so Shall I add step Fully?

File: website/package.json (hunk 6 above)
[the remark this answers is not in the document]
Comment (mskian) [reply website/package.json]: let revert the dep's and lock file

File: docs/more-pushmsg.md (hunk 1 above)
Comment (eternal-flame-AD) [reply docs/more-pushmsg.md]: To be consistent you either add a semicolon here as end of this statement, or omit all semicolons from this script.

Comment (mskian) [reply docs/more-pushmsg.md]: sorry just not clearly noticed let me change

Comment (mskian) [reply docs/more-pushmsg.md]: `npm run format` removes the semicolon

**Comment by jmattheis** [comment]
The build is currently failing because of formatting, could you run `npm run format` inside the website directory?

**Comment by mskian** [comment]
> The build is currently failing because of formatting, could you run `npm run format` inside the website directory?

yes

**Pull request merged**